app/components/travel-card.js

```diff
@@ -0,0 +1,50 @@
+import Image from "next/image"
+
+export default function TravelCard({ imageUrl, priceColor, fill, price, title, abstract }) {
+  return (
+    <div className="rounded-lg bg-white shadow-sm transition-shadow hover:shadow-md">
+      <div className="relative">
+        <Image
+          alt="Photos of works"
+          height={200}
+          src={imageUrl}
+          width={400}
+        />
+        <div
+          className={`absolute left-4 top-4 rounded-full px-3 py-1 text-xs text-white font-roboto font-black tracking-eightLetter ${priceColor}`} aria-label="blog price $49/MO"
+        >
+          {price}
+        </div>
+      </div>
+      <div className="p-6">
+        <dl>
+          <dt className="text-lightBlack font-roboto mb-2 text-xl font-black">
+            {title}
+          </dt>
+          <dd className="font-roboto mb-1 text-softGray font-normal">
+            {abstract}
+          </dd>
+        </dl>
+        <div className="flex justify-end mb-2 group" aria-label="add favorite">
```

Comment:
Como esto va a ser un objeto interactivo, mejor hacerlo desde ya un boton
```suggestion
        <button type="button" className="block ml-auto mb-2 group" aria-label="add favorite">
```